### PR TITLE
Minor improvements to SAS language support

### PR DESCRIPTION
**Pull request by cedporter**
* Reorder comment array to fix edge case added to comment_feature.test
* Adjust datalines to allow for possible preceding spaces before closing semicolon
* Move comment in sas object for correct syntax matching precedence
* Add several more keywords

### CODE REVIEW

File: components/prism-sas.js [hunk 1]
```diff
@@ -15,11 +15,11 @@
 	};
 
 	var comment = [
+		/\/\*[\s\S]+?\*\//,
```
Comment (RunDevelopment) [inline components/prism-sas.js]: I just noticed this but isn't `/**/` also a valid comment?

File: components/prism-sas.js [hunk 2]
```diff
@@ -167,7 +167,7 @@
...
 		'keyword': {
-			pattern: /((?:^|\s)=?)(?:action|after|analysis|and|array|barchart|barwidth|begingraph|by|cas|cbarline|cfill|close|column|computed?|contains|data(?=\=)|define|document|do\s+over|do|dol|drop|dul|end|entryTitle|else|endcomp|fill(?:attrs)?|filename|group(?:by)?|headline|headskip|histogram|if|infile|keep|label|layout|legendlabel|length|libname|merge|midpoints|name|noobs|nowd|ods|options|or|out(?:put)?|overlay|plot|ranexp|rannor|rbreak|retain|set|session|sessref|statgraph|sum|summarize|table|temp|then\sdo|then|title\d?|to|var|where|xaxisopts|yaxisopts|y2axisopts)\b/i,
+			pattern: /((?:^|\s)=?)(?:action|after|analysis|and|array|barchart|barwidth|begingraph|by|cas|cbarline|cfill|class(?:lev)?|close|column|computed?|contains|data(?=\=)|define|document|do\s+over|do|dol|drop|dul|end|entryTitle|else|endcomp|fill(?:attrs)?|filename|group(?:by)?|headline|headskip|histogram|if|infile|keep|keylabel|keyword|label|layout|legendlabel|length|libname|merge|midpoints|name|noobs|nowd|ods|options|or|out(?:put)?|overlay|plot|ranexp|rannor|rbreak|retain|set|session|sessref|statgraph|sum|summarize|table|temp|then\sdo|then|title\d?|to|var|where|xaxisopts|yaxisopts|y2axisopts)\b/i,
```
Comment (RunDevelopment) [inline components/prism-sas.js]: Shouldn't `then\sdo` be `then\s+do` like `do\s+over`?

File: components/prism-sas.js [hunk 3]
```diff
@@ -53,7 +53,7 @@
 
 	Prism.languages.sas = {
 		'datalines': {
-			pattern: /^(\s*)(?:(?:data)?lines|cards);[\s\S]+?^;/im,
+			pattern: /^(\s*)(?:(?:data)?lines|cards);[\s\S]+?^(\s*);/im,
```
Comment (RunDevelopment) [inline components/prism-sas.js]: 1. Please don't use capturing groups.
2. You don't need the group.

```suggestion
			pattern: /^(\s*)(?:(?:data)?lines|cards);[\s\S]+?^\s*;/im,
```

File: components/prism-sas.js (hunk 1 above)
Comment (cedporter) [reply components/prism-sas.js]: Agreed. Fixed in latest commit.

File: components/prism-sas.js (hunk 2 above)
Comment (cedporter) [reply components/prism-sas.js]: Fixed in latest commit.

File: components/prism-sas.js (hunk 3 above)
Comment (cedporter) [reply components/prism-sas.js]: Fixed in latest commit.
